src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -1947,6 +1961,7 @@ export function getChartOptions(
         viewByParentAttribute,
         stackByAttribute,
         config,
+        yAxes.length === 2,
```

Comment:
I think we should extract this to new var so that other readers can see its meaning.

Reply:
updated